### DISCUSSION TITLE
Copilot is awesome, But here are a few errors!

### DISCUSSION THREAD

**Discussion by adityapokuri229** [community/community#10955] [Copilot Conversations]
Discord.js:

// Create a command which returns the latency of the bot

**client.on('message', message => {
    // check if the message is "ping"
    if (message.content === 'ping') {
        // send a message to the channel
        message.channel.send(`${client.ping}ms`);
    }
}
);**

But `client.ping` is wrong `client.ws.ping` is correct

And Sometimes It Says **Copilot is encountering temporary issues!**

Hope My Feedback helps!

**Comment by lndmanh** [comment]
Copilot's suggestions are being made based on the user's codes and his own experiences. It does mean that if you want him to suggest using `client.ws.ping`, just delete the `client.ping` and insert the former expression several times, Copilot will automatically remember and suggest it in later coding sessions.
